src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -50,9 +51,8 @@ public AddCommand parse(String args) throws ParseException {
         AdmissionDate admissionDate =
                 ParserUtil.parseAdmissionDate(argMultimap.getValue(PREFIX_ADMISSION_DATE).orElse(null));
         Ward ward = ParserUtil.parseWard(argMultimap.getValue(PREFIX_WARD).orElse(null));
-
-
-        Person person = new Person(name, tagList, dob, ic, admissionDate, ward);
+        Remark remark = new Remark("");
+        Person person = new Person(name, tagList, dob, ic, admissionDate, ward, remark);
```

Comment:
Are we having a parseRemark() method for this? Or remarks have to be added seperately.